packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Event Hooks with Hookdeck
+excerpt: How to demonstrate Event Hooks using Hookdeck to expose a local app to the internet.
+layout: Guides
+---
+
+<StackSelector />
+
+This guide provides a working example of an Okta Event Hook. It uses the Hookdeck utility to expose a local application to the internet and receive and respond to Event Hook calls. Event Hooks are outbound calls from Okta that can notify your own software systems of events occurring in your Okta org. The Hookdeck utility enables the testing of Event Hooks locally, rather than implementing an internet-based production or test external service.
```

Comment:
```suggestion
This guide provides a working example of an Okta Event Hook. It uses the Hookdeck utility to expose a local application to the internet and receive and respond to Event Hook calls. Event Hooks are outbound calls from Okta that can notify your own software systems of events that occur in your Okta org. The Hookdeck utility enables the testing of Event Hooks locally, rather than implementing an internet-based production or test external service.
```